ruby/README.md

```diff
@@ -0,0 +1,132 @@
+# Ruby implementation of rcloadenv
+
+`rcloadenv` is a tool for loading configuration from the [Runtime Config
+API](https://cloud.google.com/deployment-manager/runtime-configurator/).
+
+This is a Ruby implementation that may be installed as a Rubygem.
+
+## Usage
+
+Install the gem using
+
+    gem install rcloadenv
+
+Alternately, include "rcloadenv" in your bundle.
+
+You may then invoke the "rcloadenv" binary. You must pass the name of the
+runtime config resource, and then the command to execute. For example:
+
+    bundle exec rcloadenv my-config -- bin/rails s
```

Comment:
This is only true if the user used a Gemfile which is stated as an alternative option. Maybe swap installation methods?